test/language/expressions/does-not-equals/bigint-and-incomparable-primitive.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Non-strict inequality comparison of BigInt and miscellaneous primitive values
+esid: sec-abstract-equality-comparison
+info: |
+  ...
+  13. Return false.
```

Comment:
this is not useful

Reply:
It would probably be more accurate to go up one level in this test's frontmatter for these particular tests --  to do this you'd need two fixes:
(1) change the esid to: sec-equality-operators-runtime-semantics-evaluation
(2) change the info to: 
`
Runtime Semantics: Evaluation
  EqualityExpression : EqualityExpression != RelationalExpression 
  ...
    5. Let r be the result of performing Abstract Equality Comparison rval == lval.
    6. If r is true, return false. Otherwise, return true. 
`
Note that your tests actually tests the "!=" equality comparison, thus the change of the esid. And the info text is a bit more clear that you are testing the "false" results of the Abstract Equality Comparison.